llvm/test/Transforms/LoopVectorize/check-no-vectorize.ll

```diff
@@ -0,0 +1,36 @@
+; This test asserts that we don't emit both
+; successful and unsuccessful message about vectorization.
+
+; RUN: opt -passes=loop-vectorize -debug -disable-output -pass-remarks-missed=loop-vectorize %s 2>&1 | FileCheck %s
```

Comment:
-debug-only=loop-vectorize? Are we checking pass remarks as well?

Reply:
Yes, thanks for catching!